### ISSUE TITLE
[Research/Audit] Dropdown - Keybindings [II]

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/spark-design-system#3584]
**User Story:**
Audit Sparks Dropdown Component so that we can see how it is currently functioning with keybindings. 

**Notes:**
Should be: https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-13
This issue is blocking: https://github.com/sparkdesignsystem/Backlog/issues/1325

**AC:**
Self audit Sparks Dropdown component.

HTML:
How do Keybindings work today?
- Review what Spark has today.
- List what needs to be updated.

ANGULAR:
How do Keybindings work today?
- Review what Spark has today.
- List what needs to be updated.

REACT:
How do Keybindings work today?
- Review what Spark has today.
- List what needs to be updated.

Follow up with PO to create Enhance the followup Issue: https://github.com/sparkdesignsystem/Backlog/issues/1325

**Comment by ssabo1** [comment]
Confirmed: Added to team backlog.
